staging/src/kubevirt.io/client-go/apis/core/v1/schema.go

```diff
@@ -499,6 +499,9 @@ type Devices struct {
 	// To configure and access client devices such as redirecting USB
 	// +optional
 	ClientPassthrough *ClientPassthroughDevices `json:"clientPassthrough,omitempty"`
+	// Whether to emulate a sound device.
+	// +optional
+	SoundDevices *SoundDevice `json:"soundDevices,omitempty"`
```

Comment:
`soundDevices` looks wrong since this is not a list. Probably `soundDevice` or maybe just `sound`?

Reply:
Is there a reason to expect that people want to add more than one sound device?

Reply:
> Is there a reason to expect that people want to add more than one sound device?

I don't think so.

> Probably soundDevice or maybe just sound?

`sound` it is.